src/core/frame/to_csv.cc

```diff
@@ -87,6 +87,17 @@ header: bool | ...
     will be written in all cases except when appending content into
     an existing file.
 
+bom: bool
+    If True, then insert the byte-order mark into the output file
```

Comment:
May be add that this is `the UTF-8 byte-order mark`?

Reply:
BOM is the unicode character U+FEFF, and this is the character that we're adding. The output file is written in UTF-8 however, so the BOM character becomes a 3-byte sequence `EF BB BF`. But this is just the consequence of the fact that we're writing a UTF-8 encoded output.